pom.xml

```diff
@@ -156,94 +174,44 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.2</version>
                 <configuration>
                     <additionalClasspathElements>
                         <additionalClasspathElement>src/test/java/</additionalClasspathElement>
                     </additionalClasspathElements>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>com.spotify</groupId>
-                <artifactId>dockerfile-maven-plugin</artifactId>
-                <version>1.4.12</version>
-                <!-- Wire up to the default build phases -->
-                <executions>
-                    <execution>
-                        <id>default</id>
-                        <goals>
-                            <goal>build</goal>
-                            <goal>push</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <repository>849905330246.dkr.ecr.us-west-2.amazonaws.com/datasonnet/datasonnet</repository>
-                    <tag>${project.version}</tag>
-                    <buildArgs>
-                        <JAR_FILE>target/${project.build.finalName}-jar-with-dependencies.jar</JAR_FILE>
-                    </buildArgs>
-                </configuration>
-                <dependencies>
-                    <!-- To make this work on JDK 9+ -->
-                    <dependency>
-                        <groupId>javax.activation</groupId>
-                        <artifactId>javax.activation-api</artifactId>
-                        <version>1.2.0</version>
-                    </dependency>
-                </dependencies>
-            </plugin>
         </plugins>
     </build>
 
     <dependencies>
         <dependency>
             <groupId>com.lihaoyi</groupId>
             <artifactId>sjsonnet_2.13</artifactId>
-            <version>0.2.4</version>
+            <version>0.2.6</version>
         </dependency>
 
         <dependency>
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
-            <version>2.13.0</version>
+            <version>2.13.3</version>
         </dependency>
 
+
         <!-- XML and CSV Plugins Dependencies -->
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>${jackson.version}</version>
+            <version>2.11.1</version>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>${jackson.version}</version>
+            <version>2.11.1</version>
```

Comment:
Why not specify Jackson version as a property so that we can easily change it in just one place, if needed?

Reply:
I can change this back. I use the maven versions plugin in order to check what's the latest on everything at once, and it can sometimes be a little finicky about versions in properties. For Jackson specifically we could consider using their BOM https://github.com/FasterXML/jackson-bom